packages/asc-ui/src/components/Footer/Footer.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
-import FooterStyle, { Props } from './FooterStyle'
+import styled from 'styled-components'
 
-const Footer: React.FC<Props> = ({ children, ...otherProps }) => (
-  <FooterStyle {...otherProps}>{children}</FooterStyle>
-)
+export interface Props extends React.HTMLAttributes<HTMLElement> {}
```

Comment:
Can we drop these props are there are no custom ones? Also perhaps we can use `HTMLFooterElement` (if it exists)?